metadata.yaml

```diff
@@ -101,6 +101,8 @@ runtimes:
   versions:
   - id: community
     name: 8.x (Community)
+  - id: community-10
```

Comment:
@lholmquist don't forget to change the version ID to match the folder name

Reply:
@gastaldi good catch